admin_interface/locale/pl/LC_MESSAGES/django.po

```diff
@@ -0,0 +1,206 @@
+# This file is distributed under the same license as the
+# django-admin-interface package.
+#
+# Translators:
+# Kamil Paduszyński <kpaduszy@gmail.com>, 2022
+#
+#, fuzzy
+msgid ""
+msgstr ""
+"Project-Id-Version: django-admin-interface\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2022-02-04 09:19+0000\n"
+"PO-Revision-Date: 2022-02-04 19:13-0500\n"
+"Last-Translator: Kamil Paduszyński <kpaduszy@gmail.com>\n"
+"Language-Team: Polish <LL@li.org>\n"
+"Language: Polish \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+
+#: admin_interface/admin.py:21
+msgid "Environment"
+msgstr "Środowisko"
+
+#: admin_interface/admin.py:30
+msgid "Language chooser"
+msgstr "Wybór wersji językowej"
+
+#: admin_interface/admin.py:37
+msgid "Logo"
+msgstr "Logo"
+
+#: admin_interface/admin.py:45
+msgid "Favicon"
+msgstr "Ikonka"
+
+#: admin_interface/admin.py:49
+msgid "Title"
+msgstr "Tytuł"
+
+#: admin_interface/admin.py:57
+msgid "Header"
+msgstr "Nagłówek"
+
+#: admin_interface/admin.py:66
+msgid "Breadcrumbs / Module headers"
+msgstr "Pasek nawigacyjny / nagłówki modułów"
+
+#: admin_interface/admin.py:76
+msgid "Generic Links"
+msgstr "Zwykłe linki"
+
+#: admin_interface/admin.py:83
+msgid "Save Buttons"
+msgstr "Przyciski zapisywania"
+
+#: admin_interface/admin.py:91
+msgid "Delete Buttons"
+msgstr "Przyciski usuwania"
+
+#: admin_interface/admin.py:99
+msgid "Related Modal"
+msgstr "Okno dialogowe"
+
+#: admin_interface/admin.py:109
+msgid "List Filter"
+msgstr "Filtry"
+
+#: admin_interface/admin.py:113
+msgid "Recent Actions"
+msgstr "Ostatnie działania"
+
+#: admin_interface/apps.py:11
+msgid "Admin Interface"
+msgstr "Wygląd panelu administracyjnego"
+
+#: admin_interface/models.py:72 admin_interface/models.py:116
+#: admin_interface/models.py:135
+msgid "name"
+msgstr "nazwa"
+
+#: admin_interface/models.py:75 admin_interface/models.py:132
+#: admin_interface/models.py:253
+msgid "active"
+msgstr "aktywny"
+
+#: admin_interface/models.py:79
+#: admin_interface/templates/admin/base_site.html:6
+#: admin_interface/templates/admin/base_site.html:60
+msgid "Django administration"
+msgstr "Administracja Django"
+
+#: admin_interface/models.py:81
+msgid "title"
+msgstr "tytuł"
+
+#: admin_interface/models.py:87 admin_interface/models.py:102
+#: admin_interface/models.py:122
+msgid "color"
+msgstr "kolor"
+
+#: admin_interface/models.py:90 admin_interface/models.py:105
+#: admin_interface/models.py:289
+msgid "visible"
+msgstr "widoczny"
+
+#: admin_interface/models.py:95
+msgid "Leave blank to use the default Django logo"
+msgstr "Puste ustawia domyślne logo Django"
+
+#: admin_interface/models.py:96
+msgid "logo"
+msgstr "logo"
+
+#: admin_interface/models.py:110
+msgid "(.ico|.png|.gif - 16x16|32x32 px)"
+msgstr "Format pliku: .ico, .png, .gif; wymiary: 16 x 16 lub 32 x 32 px).
```

Comment:
Missed the closing quote here (and the opening parenthese in the string)

Reply:
Thank you for noticing this, I already fixed it.